### ISSUE TITLE
Accessibility violation in tabs carbon component

### ISSUE THREAD

**Issue by shalaka-kshirsagar** [carbon-design-system/carbon#1046]
Working on accessibilty for Cloud Automation Manager
DAP tool used for accessibility throws a violation for 'tabs' carbon component
Violation : "The WAI-ARIA role(s) and/or attribute(s) aria-selected are not valid for the element a"

![tabs](https://user-images.githubusercontent.com/40288101/44645699-29045980-a9f6-11e8-98fc-1aadd4270923.PNG)

I ran DAP and got the violation on site: http://www.carbondesignsystem.com/components/tabs/code

![cctabs](https://user-images.githubusercontent.com/40288101/44646724-877f0700-a9f9-11e8-9539-7bb90d5c2a14.PNG)

Please let me know code fix for this. Thank you.

**Comment by tw15egan** [comment]
Unable to reproduce running latest DAP tools

**Comment by aparnacyanam** [comment]
@tw15egan It starts showing once you start tabbing and run DAP again.
This is what we are seeing right now
![image](https://user-images.githubusercontent.com/12177697/48026805-10f73780-e115-11e8-9a49-559f047e6f26.png)

**Comment by tw15egan** [comment]
cc @dakahn, something to keep an eye out for when working on https://github.com/IBM/carbon-components/issues/1308

**Comment by PrinceMinhas** [comment]
Changing the role="presentation" to role="tab" for the li elements is not a good solution. It causes the screen reader to announce either the previous or the next button along with the name of the tab that it is currently focussed on. We have to find a way to remove the aria-selected attribute from the li elements while making sure no additional problems are being created.

**Comment by dakahn** [comment]
confirmed this is still an issue in v10. Nosed around a bit and DAPs exactly right. The `<li>` should _not_ have the `aria-selected` attribute. Inspecting the code right now looks something like this:

```html
<ul>
  <li aria-selected="true">
    <a aria-selected="true"></a>
  </li>
  ...
</ul>
```

We're _somehow_ programatically setting that list item's `aria-selected` attribute to true (maybe in `_updateMenuState`) in tabs.js when only the inner anchor should be selected.

### references
https://inclusive-components.design/tabbed-interfaces/
https://www.stefanjudis.com/blog/aria-selected-and-when-to-use-it/#the-actual-use-case-of-code-aria-selected-code-

**Comment by tw15egan** [comment]
Seems to be resolved, not seeing any DAP errors and only seeing one `aria-selected`, correctly placed on the `a`

<img width="840" alt="Screen Shot 2019-07-26 at 12 01 45 PM" src="https://user-images.githubusercontent.com/11928039/61975169-47bff700-af9d-11e9-91b5-7ae60b0af5cb.png">
